.pre-commit-config.yaml

```diff
@@ -13,7 +13,9 @@ repos:
   rev: 6.1.0
   hooks:
   - id: flake8
-    additional_dependencies: ["flake8-pyproject", "flake8-logging-format", "pep8-naming"]
+    # setuptools is required by flake8-logging-format, but not declared
+    # as dependency.
```

Comment:
can you link the issue in this comment too, so that we know if we can ever drop the explicit dependency?

Reply:
Extended the comment.